tests/integration/test_keeper_client/test.py

```diff
@@ -23,10 +24,8 @@ def started_cluster():
         cluster.shutdown()
 
 
-def test_base_commands(started_cluster):
```

Comment:
Tests are not running. I think we need to keep `started_cluster` as an argument.